packages/gestures/package.json

```diff
@@ -14,8 +14,8 @@
   "scripts": {
     "start": "vite serve dev --host",
     "dev": "yarn start",
-    "build": "tsup",
-    "test": "uvu -r solid-register"
+    "build": "echo noop build",
```

Comment:
Why noop build?

Reply:
Because it was breaking, and the package is not finished, so there is no value in having the build script anyway. It would be set back to normal once the package is completed.